grate/Configuration/IFoldersConfiguration.cs

```diff
@@ -2,7 +2,7 @@
 
 namespace grate.Configuration;
 
-public interface IFoldersConfiguration: IDictionary<string, MigrationsFolder?>
+public interface IFoldersConfiguration : IDictionary<string, MigrationsFolder?>
```

Comment:
I'm not used to having i space before the colon in inheritance clauses, but I see that the standard for this is `true`, and I don't want to have a lot of special naming rules (as RoundhousE had a _lot_ of, the naming style there is quite exotic 😆 )

But, Let's go with this, as that is standard :)

csharp_space_before_colon_in_inheritance_clause

https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/csharp-formatting-options#csharp_space_before_colon_in_inheritance_clause